compiler-cli/src/export.rs

```diff
@@ -83,3 +83,12 @@ the entrypoint.sh script.
 
     Ok(())
 }
+
+pub fn hex_tarball() -> Result<()> {
+    let config = crate::config::root_config()?;
+    let data: Vec<u8> = crate::publish::build_hex_tarball(&config)?;
+
+    let path = paths::build_tarball(&config.name, &config.version.to_string());
+    println!("Creating package file '{}'", &path.display());
```

Comment:
Can we have this last and in past tense? To match the message for `export erlang-shipment`. Without the quotes around the path too.

TIL path.display, nice!